packages/docsearch-css/src/button.css

```diff
@@ -12,6 +12,7 @@
   margin: 0 0 0 16px;
   padding: 0 8px;
   user-select: none;
+  min-width: 155px;
```

Comment:
how do we know the value is 155px?

Reply:
I just inspected the element for the value:

<img width="339" alt="CleanShot 2021-07-16 at 21 08 30@2x" src="https://user-images.githubusercontent.com/1091472/125952902-77bc0f06-a375-45ae-8fe8-9f0d4dd2bd08.png">

Not a exact one, but it works in the webpack site case.

Reply:
The computed size of the component loaded is `155.359px`, I assume that's where @chenxsan also found it.

Reply:
What if a different font, placeholder or zoom level is used? I guess it's still a safe default, but I think in some cases it will still flash

Reply:
I think users could always customize it to their own use cases. We're just providing a default one which should hopefully eliminate the flash for most of the cases. For instance, the value would work for https://docusaurus.io/ as well.

Reply:
> What if a different font, placeholder or zoom level is used?

Most of these won't have effect unless the user is already overriding the provided styles. I think this could be a safe default and it is requested quite frequently.

wdyt? @Haroenv

Reply:
It's safer with the `min-` change IMO.

But I agree with Haroen. I feel like setting the constraint on `.DocSearch-Button-Keys` is better because that's ultimately what we want as constant.

Reply:
@francoischalifour I don't think it would work on `.DocSearch-Button-Keys` considering it's controlled by a state value https://github.com/algolia/docsearch/blob/1ee5354ea5026c531eb33a37d5eca8b66b7eca9d/packages/docsearch-react/src/DocSearchButton.tsx#L51

Reply:
Ah right, we can still move that check one line below to always render that element though?

Reply:
Thanks for bringing up this as I found a problem with my solution.

There're two cases depending on the `key` state:

1. a search box with the keys
<img width="171" alt="CleanShot 2021-07-21 at 16 05 27@2x" src="https://user-images.githubusercontent.com/1091472/126453976-a20ed66b-bf64-4d61-912d-ffb6cd31c24b.png">

2. a search box without the keys
<img width="119" alt="CleanShot 2021-07-21 at 16 05 07@2x" src="https://user-images.githubusercontent.com/1091472/126453931-b7369e03-0a1a-4cda-b173-15f1881ff2ed.png">


When there's no keys, the current solution would be deficient, as it results in a wide search box:
<img width="173" alt="CleanShot 2021-07-21 at 16 10 17@2x" src="https://user-images.githubusercontent.com/1091472/126454688-792c019f-480c-4b99-9698-f4e820184de6.png">

I'll see what I can do with it.